### PR TITLE
Update vit.py

### PR DESCRIPTION
**Pull request by Albdl03**
Add output port with train and validation loss and accuracy in a table format for every epoch

### CODE REVIEW

File: Extension/src/nodes/vit.py [hunk 1]
```diff
@@ -48,6 +49,9 @@
     port_type=kutil.classification_model_port_type,
     description="Output containing the trained Vision Transformer model.",
 )
+@knext.output_table(
+    name="Output table", description="Table containing results for each epoch for train and validation."
```
Comment (dsaam94) [inline Extension/src/nodes/vit.py:53]: Rename, `Output table` to something better

File: Extension/src/nodes/vit.py [hunk 2]
```diff
@@ -67,7 +71,7 @@ class VisionTransformerLearnerNode:
         label="Label Column",
         description="Select the column containing class labels.",
         port_index=0,
-        column_filter=kutil.is_nominal,
+        column_filter=kutil.is_numeric_or_string,
```
Comment (dsaam94) [inline Extension/src/nodes/vit.py:74]: `is_nominal` is fine

File: Extension/src/nodes/vit.py (hunk 1 above)
Comment (dsaam94) [reply Extension/src/nodes/vit.py:53]: Also add more details about the loss function (maybe a link) for end-user to understand more about the metrics.

**Review by dsaam94 — CHANGES_REQUESTED** [review]
